openslides/utils/autoupdate.py

```diff
@@ -40,7 +40,7 @@ def ws_add_site(message):
             # Skip apps that do not implement get_startup_elements
             continue
         for collection in get_startup_elements():
-            output.extend(collection.as_autoupdate_for_user(message.user))
+            output.extend(collection.as_autoupdate_for_user(message.user.id))
```

Comment:
Ok. It was not self explaining that `user` as namend in the method wokrs the same as `user.id` as given by this line. So I would appreciate you to add a comment or docstring extension or thelike (and if you are in good mood a unit test for the as_autoupdate_for_user method called with an id and None). :-)